webapplication/sip/settings.py

```diff
@@ -20,7 +20,11 @@
 # See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'bnc-3!@tium@48wg*d(bdqqczssab=ykl2$30-%*%w9g3&945*'
+DEFAULT_SECRET_KEY = 'bnc-3!@tium@48wg*d(bdqqczssab=ykl2$30-%*%w9g3&945*'
+if os.getenv("SECRET_KEY"):
+    SECRET_KEY = os.getenv("SECRET_KEY")
+else:
+    SECRET_KEY = DEFAULT_SECRET_KEY
```

Comment:
SECRET_KEY = os.getenv("SECRET_KEY", 'bnc-3!@tium@48wg*d(bdqqczssab=ykl2$30-%*%w9g3&945*')

Reply:
I've started with this solution, and it doesn't work   
that's why I'm not sure in it:

because of this 
```
ARG SECRET_KEY
ENV SECRET_KEY ${SECRET_KEY}
```
while we are creating container with this command:
```
docker build -t registry.quantumobile.co/portal_soilmate_webapplication -f ./webapplication/prod.Dockerfile \
./webapplication
```
 and NOT passing the new value for SECRET_KEY
 in container we have envs like this
```
         "Env": [
            "DJANGO_SETTINGS_MODULE=sip.settings-lviv",
            "POSTGRES_DB=sip_db",
            "POSTGRES_USER=sip",
            "POSTGRES_PASSWORD=eiJ2koo6",
            "PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
            "LANG=C.UTF-8",
            "GPG_KEY=0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D",
            "PYTHON_VERSION=3.7.7",
            "PYTHON_PIP_VERSION=20.1.1",
            "PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/eff16c878c7fd6b688b9b4c4267695cf1a0bf01b/get-pip.py",
            "PYTHON_GET_PIP_SHA256=b3153ec0cf7b7bbf9556932aa37e4981c35dc2a2c501d70d91d2795aa532be79",
            "PYTHONUNBUFFERED=1",
            "SECRET_KEY="
        ],
```
       in python part it become SECRET_KEY="" and we are not using default value, we are using empty
  this is correct?